x/leveragelp/keeper/position.go

```diff
@@ -97,9 +92,15 @@ func (k Keeper) DestroyPosition(ctx sdk.Context, positionAddress string, id uint
 		if len(stopLossKey) > 0 {
 			store.Delete(stopLossKey)
 		}
-		store.Delete([]byte(old.GetPositionAddress()))
+		store.Delete(old.GetPositionAddress())
```

Comment:
```suggestion
		store.Delete([]byte(old.GetPositionAddress()))
```

Reply:
I think this should be bytes, as we store key in bytes

Reply:
AccAddress is always in Byte format

Reply:
Though now I am unsure what it's deleting? No prefix key was used. Ideally it should have types.GetPositionKey to delete, but now that is being called in next line

Reply:
makes sense.
This will be removed completely as we won't be using sorted maps.

Reply:
Shall I remove this code then?

Reply:
not yet, it will be part of `new-leverage-flow` branch and we will merge to `main` after everything looks good with new flow